src/conversion.rs

```diff
@@ -244,25 +244,77 @@ where
     }
 }
 
-/// Extract reference to instance from `PyObject`
-impl<'a, T> FromPyObject<'a> for &'a T
-where
-    T: PyTryFrom<'a>,
-{
-    #[inline]
-    fn extract(ob: &'a PyAny) -> PyResult<&'a T> {
-        Ok(T::try_from(ob)?)
+pub mod extract_impl {
```

Comment:
I checked `extract_impl` doesn't appear on the rust doc.
If it is intended, this should be marked as `#[doc(hidden)]`

Reply:
👍